.github/workflows/integration.yml

```diff
@@ -58,17 +64,111 @@ jobs:
         run: |
           echo "${GITHUB_REF#refs/heads/}"
           git ls-remote --exit-code --heads https://github.com/infrawatch/sg-bridge.git "$(echo ${GITHUB_REF#refs/heads/})"
-      - name: Start sg-bridge for collectd from container image
+      - name: Start sg-bridge from container image
         if: steps.bridge_branch.outcome != 'success'
         run: |
           docker run --name=sgbridge --network host $BRIDGE_VOLUME -d \
-            $BRIDGE_IMAGE --amqp_url amqp://localhost:5666/$QDR_CHANNEL_COLLECTD \
+            $BRIDGE_IMAGE --amqp_url amqp://localhost:5666/$QDR_CHANNEL \
             --gw_unix=$BRIDGE_SOCKET
-      - name: Start sg-bridge for collectd with same branch
+      - name: Start sg-bridge for with same branch
         if: steps.bridge_branch.outcome == 'success'
         run: |
           docker run --name=sgbridge --network host $BRIDGE_VOLUME -d -uroot \
-            -e GITHUB_REF -e BRIDGE_SOCKET -e QDR_CHANNEL_COLLECTD -e OPSTOOLS_REPO \
+            -e GITHUB_REF -e BRIDGE_SOCKET -e QDR_CHANNEL -e OPSTOOLS_REPO \
+            --workdir=$(dirname $BRIDGE_SOCKET) \
+            $TEST_IMAGE bash $PROJECT_ROOT/ci/integration/metrics/run_bridge.sh
+      - name: Install collectd
+        run: |
+          sudo apt-get install collectd
```

Comment:
What version of collectd does this install?

Reply:
I see now you've just moved what was already happening up here. Still curious, but this seems fine :)

Reply:
I don't have ubuntu on hand right now, but I found this: https://www.ubuntuupdates.org/package/core/jammy/universe/updates/collectd So I guess it could be 5.12?